src/components/color-control/paletteControl.js

```
@@ -38,9 +39,50 @@ const ColorPaletteControl = props => {
+		// If that fails, try the direct selector
+		if (!colors || colors.length === 0) {
+			colors = receiveMaxiSelectedStyleCardValue('customColors') || [];
+		}
+
+		// If still no colors, try to get the styleCard directly
+		if (!colors || colors.length === 0) {
+			const styleCard = select(
+				'maxiBlocks/style-cards'
+			).receiveMaxiSelectedStyleCard();
+
+			if (styleCard && styleCard.value) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)